src/main/java/core/basesyntax/utils/generate/impl/csv/CsvGeneratorImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.utils.generate.impl.csv;
+
+import core.basesyntax.storage.impl.StorageImpl;
+import core.basesyntax.utils.generate.ReportGenerator;
+
+public class CsvGeneratorImpl implements ReportGenerator {
+    private static final String LINE_SEPARATOR = ",";
+    private static final String REPORT_INIT = "fruit,quantity";
+
+    @Override
+    public String getReport(StorageImpl storage) {
```

Comment:
Avoid hardcoding the separator and report initialization string directly in the class. These could be passed as parameters or configured externally to make the class more flexible.